sql/_13_issues/_23_1h/cases/cbrd_24843_2.sql

```diff
@@ -0,0 +1,647 @@
+/* ----------------------------------------
+ *
+ * test case 36:
+ *
+ * 1. 'c.p_a = 1' -> pt_lambda_node() -> 'p.a = 1' [o]
+ * 2. 'p.a = 2' -> pt_lambda_node() -> 'c.p_a = 2' [o]
+ * 3. 'p.a = 3 and p.b = 3' -> pt_lambda_node() -> 'c.p_a = 3 and c.p_b = 3' [o]
+ * 4. 'p.a > 3 and p.a = 4' -> pt_lambda_node() -> 'c.p_a = 4' [o]
+ * 5. '(c.p_a = 5 or c.p_b = 5)' [o]
+ * 6. '(p.a = 6 or p.b = 6)' [x]
```

Comment:
from @youngjinj 

If have reference of 'parent p', cannot eliminate parent.
In case of equality (=) condition, you can make the condition on parent into a condition on child by join condition.
But In case as 6, 8, 9, 10 , scope conditon cannot make condition of parent into a condition on child.